client/source/js/modules/analysis/optimization-ctrl.js

```diff
@@ -9,6 +9,7 @@ define([
 
       $scope.meta = meta;
       $scope.types = angular.copy(CONFIG.GRAPH_TYPES);
+      $scope.optimizationStatus = false;
```

Comment:
"false" makes it easier to check. Otherwise I'd prefer 0-based enums, not sure if they exist in Javascript?